assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -498,3 +498,8 @@ che.workspace.feature.api=NULL
 #   - 'jwtproxy': jwtproxy will authenticate requests.
 #       So, servers will receive only authenticated ones.
 che.server.secure_exposer=default
+
+# Jwtproxy issuer string, token lifetime and optional auth page path to route unsigned requests to.
+jwtproxy.token.issuer=wsmaster
```

Comment:
I would say that properties names do not meet Che properties naming convention.
I think it would be better to use prefix something like `che.server.secure_exposer.jwtproxy`